connectors/http/rest/element-templates/http-json-connector.json

```diff
@@ -333,14 +333,14 @@
     },
     "type" : "Dropdown",
     "choices" : [ {
+      "name" : "DELETE",
+      "value" : "DELETE"
+    }, {
```

Comment:
Can we move `DELETE` down the list where it was before? We have a nice new order property :)

Reply:
Not sure, why this was changed. I reverted it.